libindy/src/api/ledger.rs

```diff
@@ -1792,6 +1792,91 @@ pub extern fn indy_get_response_metadata(command_handle: CommandHandle,
     res
 }
 
+/// Builds a LEDGERS_FREEZE request. Request to freeze list of ledgers.
+///
+/// #Params
+/// command_handle: command handle to map callback to caller context.
+/// submitter_did: (Optional) DID of the read request sender (if not provided then default Libindy DID will be used).
+/// ledgers_ids: list ids for freeze ledgers (json format).
+/// cb: Callback that takes command result as parameter.
+///
+/// #Returns
+/// Request result as json.
+///
+/// #Errors
+/// Common*
+#[no_mangle]
+pub extern fn indy_build_ledgers_freeze_request (command_handle: CommandHandle,
+                                                 submitter_did: *const c_char,
+                                                 ledgers_ids: *const c_char,
```

Comment:
Is it `c_char` or `Vec<u64>`? 
If it's a Vec, please update the description comment.